src/code.sh

```
@@ -81,150 +44,326 @@ _trim_fastq_endings () {
+              echo "$sub_job_tars" | xargs -P $IO_PROCESSES -n1 -I{} sh -c "dx cat $DX_WORKSPACE_ID:{} | tar xf - -C inputs_${fusion%.*}"
+       done
 
-comm -13 ${out_filename}_fusion_contigs_inspected.txt fusion_rescue_list.txt > ${out_filename}_missed_fusion_contigs.txt
 
-mark-section "Move results files to their output directories"
+       mark-section "Generating the html report containing all the fusions detected"
+       mkdir combined_files
 
-find /home/dnanexus -type f -name ${out_filename}_fusion_contigs_inspected.txt -printf "%f\n" | \
-xargs -I{} mv /home/dnanexus/{} /home/dnanexus/out/fi_inspected_fusions/{}
+       cat $(find . -type f -name "cytoBand.txt") > combined_files/cytoBand.txt
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)